liver_imaging_analysis/engine/engine.py

```diff
@@ -444,7 +527,10 @@ def test(self, dataloader = None, callback = False):
                     batch[Keys.LABEL]
                     ).item()
                 #Apply post processing transforms on prediction
+                pre_post_processing = time.time()
                 batch = self.post_process(batch)
+                post_post_processing = time.time()-pre_post_processing
+                post_processing_time += post_post_processing
```

Comment:
same here please take the time debugging out

Reply:
done